spec/observables/dom/ajax-spec.ts

```diff
@@ -823,11 +810,10 @@ describe('ajax', () => {
     });
 
     it('should succeed on 204 No Content', () => {
-      const expected: null = null;
       let result: AjaxResponse<any>;
       let complete = false;
 
-      ajax.post('/flibbertyJibbet', expected).subscribe(
```

Comment:
no idea why I chose to post a body of `null` here, seems weird. But this whole test was really just to check that 204 status passed okay.